lib/nationbuilder/client.rb

```diff
@@ -44,6 +45,8 @@ def raw_call(path, method, body = {}, args = {})
       }
     }
 
+    request_args[:query][:fire_webhooks] = @fire_webhooks unless @fire_webhooks.nil?
```

Comment:
Gotcha. In our case, we're trying to push data into NB without it coming back to us via webhooks, so we'll be disabling webhooks in any updates to people or donations. It might be worth scraping the pull request if you don't want to handle it this way.